src/Endpoints/user-meta/class-editor-sidebar-settings-endpoint.php

```diff
@@ -41,6 +41,22 @@ protected function get_meta_key(): string {
 	 */
 	protected function get_subvalues_specs(): array {
 		return array(
+			'CrossLinksMaxLinkWords'       => array(
+				'values'  => array(),
+				'default' => 4,
+			),
+			'CrossLinksMaxLinks'           => array(
+				'values'  => array(),
+				'default' => 10,
+			),
```

Comment:
We still have 3 occurrences of default values in PHP files. You can leave as-is if you think this is fine.